databuilder/databuilder/extractor/redshift_table_owner_extractor.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from databuilder.extractor.base_postgres_owner_extractor import BasePostgresOwnerExtractor
+
+
+class RedshiftTableOwnerextractor(BasePostgresOwnerExtractor):
+    """
+    Extracts Redshift table usage stats from underlying meta store database using SQLAlchemyExtractor
+    This differs from the PostgresMetadataExtractor because in order to support Redshift's late binding views,
+    we need to join the INFORMATION_SCHEMA data against the function PG_GET_LATE_BINDING_VIEW_COLS().
+    """
+
+    def get_sql_statement(self, use_catalog_as_cluster_name: bool, where_clause_suffix: str) -> str:
+        if use_catalog_as_cluster_name:
+            cluster_source = "CURRENT_DATABASE()"
+        else:
+            cluster_source = f"'{self._cluster}'"
+
+        return """
+        SELECT
+        'redshift' as database
+         ,n.nspname AS schema
+         , c.relname AS table_name
+         , pg_get_userbyid(c.relowner) AS owner
```

Comment:
This is a very opinionated approach to getting the owner. How can this be extended to allow others to easily configure the way in which the owner is retrieved?